VideoWeb/VideoWeb/ClientApp/src/app/services/logging/logger.service.ts

```diff
@@ -8,25 +9,52 @@ export const LOG_ADAPTER = new InjectionToken<LogAdapter>('LogAdapter');
     providedIn: 'root'
 })
 export class LoggerService implements Logger {
+    static currentConferenceIdPropertyKey = 'currentConferenceId';
+
+    public conferenceService: ConferenceService;
     constructor(@Inject(LOG_ADAPTER) private adapters: LogAdapter[]) {}
 
     debug(message: string, properties?: any): void {
+        if (typeof properties === 'object') {
+            properties = properties ?? {};
+            properties[LoggerService.currentConferenceIdPropertyKey] = this.conferenceService?.currentConferenceId;
+        }
+
         this.adapters.forEach(logger => logger.debug(message, properties));
     }
 
     info(message: string, properties?: any): void {
+        if (typeof properties === 'object') {
+            properties = properties ?? {};
+            properties[LoggerService.currentConferenceIdPropertyKey] = this.conferenceService?.currentConferenceId;
+        }
+
         this.adapters.forEach(logger => logger.info(message, properties));
     }
 
     warn(message: string, properties?: any): void {
+        if (typeof properties === 'object') {
```

Comment:
very minor, but we can extract this logic out into a function and use it instead of the code duplication.

Reply:
That is a very good point, will do it now :D